frontend/src/pages/JobPage/JobPage.jsx

```diff
@@ -32,6 +32,13 @@ class JobPage extends React.Component {
         status: PropTypes.object.isRequired
     };
 
+    constructor(props) {
+        super(props);
+        this.state = {
+            autoScrollToBottom: true,
```

Comment:
is there a performance hit for this? that might mean we want to default to `false`?

Reply:
I can't think of why there would be a performance hit.